backend/inspirehep/records/api/base.py

```diff
@@ -443,6 +445,15 @@ def linked_institutions_pids(self):
             )
         )
 
+    @property
+    def linked_experiments_pids(self):
+        return set(
+            chain.from_iterable(
```

Comment:
since it only uses one field, you don't need all of this here. You can simply return the ```get_linked_pids_from_field``` of that field